src/AutoLaunchWindows.coffee

```diff
@@ -52,8 +58,11 @@ module.exports =
 
 
     # appName - {String}
+    # onlyMe - (Optional) {Boolean}
```

Comment:
A comment for `mac` is missing

Reply:
Please, use a name that is more representative of what purpose of the parameter. We usually talk about a user-wide or a system-wide installation (something like "systemWideInstall").